### PR TITLE
Fix mentions of specific releases

### PR DESCRIPTION
**Pull request by geeknoid**
- Remove the release-specific wording on the main release note page and on the docs page. This
ends up being hard to keep correct and not really useful.

- Add a full_version variable in args.yml which contains the 3 part release version
such as 1.0.1. When we release a new patch, we need to update this number in the current
release branch.

- Apply the full_version variable to the download button on the home page. It will now say "DOWNLOAD 1.0.1".

### CODE REVIEW

File: content/about/notes/_index.md [hunk 1]
```diff
@@ -19,19 +19,3 @@ icon: /img/notes.svg
 - [Istio 0.3](./0.3)
 - [Istio 0.2](./0.2)
 - [Istio 0.1](./0.1)
-
```
Comment (hklai) [inline content/about/notes/_index.md]: It may be useful to keep a link back to https://github.com/istio/istio/releases?

File: data/args.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 version: "1.1"
+full_version: "1.1.0"
```
Comment (hklai) [inline data/args.yml]: Is this supposed to be 1.0.1?

**Comment by hklai** [comment]
Also, is somebody else going to take care of content_zh?

**Comment by geeknoid** [comment]
You can preview the changes by clicking on the deploy/netlify link.

This change is being applied to master. When I cherry-pick it to the release-1.0 branch, I'll change the full_version string to 1.0.1

Our industrious Chinese translation crew take care of the content_zh stuff separately.

File: content/about/notes/_index.md (hunk 1 above)
Comment (geeknoid) [reply content/about/notes/_index.md]: Done.

**Pull request merged**